packages/camel-catalog/catalog-generator/src/main/java/io/kaoto/camelcatalog/KameletProcessor.java

```diff
@@ -0,0 +1,39 @@
+package io.kaoto.camelcatalog;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.util.List;
+
+public class KameletProcessor {
+    private static final List<String> TO_STRING_TYPES = List.of("binary");
+
+    public static void process(ObjectNode kamelet) {
+        var schema = kamelet.withObject("/propertiesSchema");
+        var kameletDef = kamelet.withObject("/spec")
+                .withObject("/definition");
+        schema.put("$schema", "http://json-schema.org/draft-07/schema#");
```

Comment:
What is teh reason to not use the latest schema version?

https://json-schema.org/draft/2020-12/schema

Reply:
At this moment, this is the version supported by our version of `ajv`